src/utilities.cpp

```diff
@@ -556,4 +559,122 @@ void findAllVariablesWithEquivalences(const ComponentPtr &component, VariablePtr
     }
 }
 
+IdList listIds(const ModelPtr &model, bool mathIds)
```

Comment:
seems to be very similar code here and in the annotator...any way to merge them?